README.md

```diff
@@ -503,3 +504,19 @@ total_failures = wavefront_sender.get_failure_count()
   the minor and patch versions.
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## How to Release
+
+1. Merge all the changes that need to go into the release to the master branch.
+2. Open the `setup.py` file from the top level directory of the project.
+3. Search for `version=` in the file to find the version number for example `1.8.10`.
+4. Log into Github, click **Releases** on the right, and click **Draft a new release.**
+5. For **Choose a tag**, choose the version you found in step 3, and prefix it with `v` for example `v1.8.10`. You will need to enter the version where it says **Find or create new tag.**
```

Comment:
You need to enter the version where it says **Find or create new tag**.

Reply:
Done.